### PR TITLE
[zh-cn] Fixes Broken Link

### LINKED ISSUE #36910: [zh-cn] Need to update the `code-contribution.md` with full link
**This is a Bug Report**
I have fixed the same issue in English docs, same issue is followed by the other localization team. [This page](https://kubernetes.io/zh-cn/docs/reference/glossary/?fundamental=true&community=true) has one broken issue which needs to fix.

china

Refer my merged PR of english docs.
 [kubernetes/website](https://github.com/kubernetes/website/pull/36881)

/assign

### PR DESCRIPTION
**Pull request by amolmote**
Signed-off-by: amolmote <amolmote201@gmail.com>

Fixes #36910

### CODE REVIEW

**Comment by Sea-n** [comment]
I think it's better to fix `content/en/docs/reference/glossary/code-contributor.md` instead.

**Comment by amolmote** [comment]
I already fix it

**Comment by Sea-n** [comment]
Oh, sorry for my local branch missed PR #36881.

You are right, you already fixed this in English version.

**Comment by Sea-n** [comment]
Please update line 18 as well, thanks!

**Comment by tengqm** [comment]
@amolmote Please consider address the feedback.

**Comment by amolmote** [comment]
Can I remove the unwanted comments because it's has no use

**Comment by tengqm** [comment]
@amolmote Line 18 should be sync'ed with the English version as well. Although it is in comments. We compare these commented out paragraphs against the English version to find diffs.

**Comment by amolmote** [comment]
Ok will push the new changes by squashing commits.

**Comment by amolmote** [comment]
done @tengqm

**Review by Sea-n — COMMENTED** [review]
/lgtm

File: content/zh-cn/docs/reference/glossary/code-contributor.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ tags:
 title: Code Contributor
 id: code-contributor
 date: 2018-04-12
-full_link: /docs/community/devel/
+full_link: https://github.com/kubernetes/community/tree/master/
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/code-contributor.md:18]: Line 18 should be same as line 5, right?

**Comment by amolmote** [comment]
Yes absolutely @tengqm, my bad correct it asap.

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**